### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
List of changes:
- Set readme as default html doc page
- Badges and more badges!
- Fix typos
- Use common source url
- Remove unused hello world function and test
- Use indentation as code block marker
- Rename and add testing doc
- Add changelog file and link in hex info page

### CODE REVIEW

File: lib/control_node.ex [hunk 1]
```diff
@@ -2,17 +2,4 @@ defmodule ControlNode do
...
-
-  ## Examples
-
-      iex> ControlNode.hello()
-      :world
-
-  """
-  def hello do
-    :world
-  end
```
Comment (kansi) [inline lib/control_node.ex:17]: Please remove the corresponding test too!

**Review by kansi — COMMENTED** [review]
@kianmeng thank you for the PR :slightly_smiling_face: 

Please address the comment

File: lib/control_node.ex (hunk 1 above)
Comment (kianmeng) [reply lib/control_node.ex:17]: > Please remove the corresponding test too!

@kansi Done.

File: README.md [hunk 2]
```diff
@@ -115,10 +116,10 @@ AllowTcpForwarding yes
 ...
 ```
 
-## Limiation
+## Lamination
```
Comment (kansi) [inline README.md]: Should be `Limitations`

File: README.md [hunk 3]
```diff
@@ -115,10 +116,10 @@ AllowTcpForwarding yes
...
 
-- Only shortnames for nodes are allowed
+- Only short names for nodes are allowed
 - Instances of same release (deployed to different) should have different
   hostname i.e. for eg. if node 1 has node name `service_app@host1` then another
   node of `service_app` should have a different node name.
-- SSH client cannot read new format of RSA keys, should be convered to PEM format
+- SSH client cannot read new format of RSA keys, should be covered to PEM format
```
Comment (kansi) [inline README.md]: ~covered~ converted

**Comment by kansi** [comment]
Looks good, thank you for the contribution :slightly_smiling_face:

**Pull request merged**